src/pages/guidelines/accessibility/color.mdx

```diff
@@ -20,7 +21,7 @@ tabs: ['Overview', 'Color', 'Keyboard', 'Developers']
 <ResourceCard
     subTitle="Carbon color contrast tool"
     href="https://marijohannessen.github.io/color-contrast-checker/"
-    
+
```

Comment:
build error seems to have something to do with this line break. I would try just updating it to

```
    <ResourceCard
      subTitle="Carbon color contrast tool"
      href="https://marijohannessen.github.io/color-contrast-checker/"
    />
```